### PR TITLE
cloud-init: add 2023q2 report

### PR DESCRIPTION
**Pull request by igalic**
In this quarter, I've delivered a new milestone.
and some other exciting updates.

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/cloud-init.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+These are used by several cloud providers to initialize a temporary network before retrieving the actual configuration.
+
+cloud-init has been successfully tested on Vultr, and I hope that with the next release I can convince Vultr to switch their FreeBSD images to cloud-init.
+
+In addition to that, I have expanded rsyslog support for BSD.
+I've also added an rc script for cloud-init's ds-identify, which should make zero-configuration setups orders of magnitude faster:
+ds-identify runs first and very quickly guesses the cloud provider it is running on.
+
+The next steps will be keep hacking away at the network refactoring tasks, and add LXD support for FreeBSD, so it can be included in CI tests.
+The latter will include link:https://github.com/canonical/lxd/pull/11761[work on LXD], as well as work on the link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=271793[FreeBSD virtio subsystem].
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/cloud-init.adoc:27]: Consistency: I would either include or exclude the word "work" from both pairs of square brackets.

File: website/content/en/status/report-2023-04-2023-06/cloud-init.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This quarter has been going quite slowly, but I have managed to deliver a new milestone:
+
+- Ephemeral Networking classes have been rewritten and made platform independent.
+  These are used by several cloud providers to initialize a temporary network before retrieving the actual configuration.
+
+- cloud-init has been successfully tested on Vultr.
+  I hope that with the next release I can convince Vultr to switch their FreeBSD images to cloud-init.
+
+In addition to that, I have expanded rsyslog support for BSD.
```
Comment (igalic) [inline website/content/en/status/report-2023-04-2023-06/cloud-init.adoc:23]: slightly improved wording here

File: website/content/en/status/report-2023-04-2023-06/cloud-init.adoc (hunk 1 above)
Comment (igalic) [reply website/content/en/status/report-2023-04-2023-06/cloud-init.adoc]: done

**Review by igalic — COMMENTED** [review]
made some improvements guided by @lsalvadore's suggestions

**Pull request closed without merging**